<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentDlg</class>
 <widget class="QDialog" name="SegmentDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Add route:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="rbNoAdd">
           <property name="toolTip">
            <string>No route</string>
           </property>
           <property name="statusTip">
            <string>Don't add the segments to a route.</string>
           </property>
           <property name="text">
            <string>Don't add</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbUseNew">
           <property name="toolTip">
            <string>Add new segment to route</string>
           </property>
           <property name="statusTip">
            <string>Add the new portion to the route</string>
           </property>
           <property name="text">
            <string>Use new</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbUseOriginal">
           <property name="toolTip">
            <string>Add the original segment to the route</string>
           </property>
           <property name="statusTip">
            <string>The route uses the original segment. </string>
           </property>
           <property name="text">
            <string>keep original</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>rbUseNew</zorder>
        <zorder>rbNoAdd</zorder>
        <zorder>rbUseOriginal</zorder>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox7">
        <property name="title">
         <string>Route Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Company</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="RouteNameWidget" name="rnw" native="true"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Start date:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QDateEdit" name="dateStart">
           <property name="displayFormat">
            <string>yyyy/MM/dd</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>End Date:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QDateEdit" name="dateEnd">
           <property name="displayFormat">
            <string>yyyy/MM/dd</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3" rowspan="2">
          <widget class="QGroupBox" name="groupBox2">
           <property name="title">
            <string>Direction:</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="rbNormal">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbReverse">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Traction type:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="cbTractionType"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox3">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Normal Enter:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="rbNFromLeft">
              <property name="text">
               <string>From left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbNfromBack">
              <property name="text">
               <string>From back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbNFromRight">
              <property name="text">
               <string>FromRight</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QGroupBox" name="groupBox4">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Normal leave:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="rbNtoLeft">
              <property name="text">
               <string>To left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbNAhead">
              <property name="text">
               <string>Ahead</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbNToRight">
              <property name="text">
               <string>To right</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QGroupBox" name="groupBox5">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Reverse enter:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QRadioButton" name="rrbRFromLeft">
              <property name="text">
               <string>From left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbRFromBack">
              <property name="text">
               <string>From back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="RBRFromRight">
              <property name="text">
               <string>From right</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QGroupBox" name="groupBox6">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Reverse Leave:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="rbRToLeft">
              <property name="text">
               <string>To left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbRAhead">
              <property name="text">
               <string>Ahead</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbRToRight">
              <property name="text">
               <string>To right</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QComboBox" name="cbCompany"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbOriginal">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>OriginalName:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkOriginalOneWay">
        <property name="text">
         <string>One way</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="txtNewerName_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="lbl_newerName_1">
        <property name="text">
         <string>Newer Name: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblOriginalLocation">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="EditSegmentDescr" name="txtOriginalName">
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="txtOriginalLocation"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblErrorText">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>159</red>
           <green>158</green>
           <blue>158</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>error...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbNew">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>New name:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkNewOneWay">
        <property name="text">
         <string>One Way</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tracks:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QSpinBox" name="sbTracks">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="cbLocation">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="6" rowspan="2" colspan="2">
       <widget class="QGroupBox" name="gbUsage">
        <property name="title">
         <string>Usage</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbUseL">
           <property name="text">
            <string>L</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbBoth">
           <property name="text">
            <string>Both</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbUseR">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QComboBox" name="cbRouteType"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblNewLocation">
        <property name="text">
         <string>Location: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtNewLocation"/>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QLabel" name="lbl_newerName_2">
        <property name="text">
         <string>Newer Name: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="5" colspan="3">
       <widget class="QLineEdit" name="txtNewerName_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Route Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="7">
       <widget class="EditSegmentDescr" name="txtNewName">
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditSegmentDescr</class>
   <extends>QLineEdit</extends>
   <header>editsegmentdescr.h</header>
  </customwidget>
  <customwidget>
   <class>RouteNameWidget</class>
   <extends>QWidget</extends>
   <header>routenamewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtOriginalName</tabstop>
  <tabstop>chkOriginalOneWay</tabstop>
  <tabstop>txtNewName</tabstop>
  <tabstop>chkNewOneWay</tabstop>
  <tabstop>sbTracks</tabstop>
  <tabstop>cbRouteType</tabstop>
  <tabstop>rbNoAdd</tabstop>
  <tabstop>rbUseNew</tabstop>
  <tabstop>rbUseOriginal</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
